From c6c4f5cb74eee81441864897f3249e55fe19c24c Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 4 Dec 2024 17:54:18 +0800
Subject: [PATCH 208/301] AOSCOS: input: atkbd: correct dependency for
 KEYBOARD_ATKBD_LEMOTE_KEYCODES

Rename dependency MACH_LOONGSON => MACH_LOONGSON64 per commit
30ad29bb4888 ("MIPS: Loongson: Naming style cleanup and rework") from the
v4.2 cycle.

Fixes: "AOSCOS: MIPS: Loongson: Add the multifunction keys (Fnkey) support"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/input/keyboard/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/keyboard/Kconfig b/drivers/input/keyboard/Kconfig
index 2678b3132036..dd127dac0b78 100644
--- a/drivers/input/keyboard/Kconfig
+++ b/drivers/input/keyboard/Kconfig
@@ -158,7 +158,7 @@ config KEYBOARD_ATKBD_RDI_KEYCODES
 
 config KEYBOARD_ATKBD_LEMOTE_KEYCODES
 	bool "Use Lemote laptop (A1201) keyboard scancodes"
-	depends on MACH_LOONGSON && KEYBOARD_ATKBD
+	depends on MACH_LOONGSON64 && KEYBOARD_ATKBD
 	default y
 	help
 	  Say Y here if you want to use an AT or PS/2 keyboard, and your
-- 
2.48.1

